### DISCUSSION TITLE
Is there a way to return a redirect response from the server?

### DISCUSSION THREAD

**Discussion by gdsmith** [serverpod/serverpod#680] [Q&A] [answered]
I know it is possible to return a redirect from the web routes, but not sure how to do the same for an endpoint.

**Comment by fischerscode** [comment, marked as answer]
I think there is currently no way to achieve this. But it might be a nice feature.

**Comment by vlidholt** [comment]
Can you describe a use case for this? I would be like you are calling a method in your app, but it redirects to another method?
